plasma_framework/test/src/exits/payment/spendingConditions/PaymentSpendingConditionRegistry.test.js

```diff
@@ -0,0 +1,84 @@
+const PaymentSpendingConditionRegistry = artifacts.require('PaymentSpendingConditionRegistry');
+const DummyPaymentSpendingCondition = artifacts.require('PaymentSpendingConditionTrue');
+
+const { constants, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+contract('PaymentSpendingConditionRegistry', ([_, other]) => {
+    beforeEach(async () => {
+        this.registry = await PaymentSpendingConditionRegistry.new();
+        this.dummyCondition = await DummyPaymentSpendingCondition.new();
+    });
+
+    describe('spendingConditions', () => {
+        it('should receive empty contract address when pair (outputType, spendingTx) not registered', async () => {
```

Comment:
What's the benefit of returning zero address over reverting?

Reply:
hmmm....it is default behavior and we don't need to write extra `require(...)` for that. Also this is consistent with any other registry with single value as key